MNT: Re-rendered with conda-build 3.18.2, conda-smithy 3.3.4, and con…

ver228 opened the pull request:
…da-forge-pinning 2019.05.31

<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

ver228 committed:
MNT: Re-rendered with conda-build 3.18.2, conda-smithy 3.3.4, and conda-forge-pinning 2019.05.31

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

ver228 committed:
MNT: Re-rendered with conda-build 3.18.2, conda-smithy 3.3.4, and conda-forge-pinning 2019.05.31

ver228 committed:
Update meta.yaml

- gcc as dependency (seems to be needed for conda-forge docker).
- restrict the build to python 3.6. This is due to the required keras version. I am planning to replace keras to pytorch in the next tierpsy version...

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found some lint.

Here's what I've got...


For **recipe**:

 * There are too few lines.  There should be one empty line at the end of the file.

ver228 committed:
MNT: Re-rendered with conda-build 3.18.2, conda-smithy 3.3.4, and conda-forge-pinning 2019.05.31

ver228 committed:
Update meta.yaml

add empyt line at the end as required by the linter

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

ver228 committed:
Update meta.yaml

ver228 committed:
Update meta.yaml

ver228 committed:
run conda smithy update-cb3

ver228 committed:
update opencv linux dependency

ver228 committed:
Update meta.yaml

ver228 committed:
update

ver228 committed:
Create win_c_compilervs2008.yaml

ver228 committed:
Update meta.yaml
